lib/galaxy/tool_util/ontologies/biotools_mappings.tsv

```diff
@@ -1,5 +1,1927 @@
 # tool-id <tab> bio.tools-id
 abyss-pe	abyss
 allegro	allegro
-# testing tool mapping for test case
```

Comment:
@astrovsky01 that looks like it is needed for a test case ;)

Reply:
I guess the CI is failing because that line was added back in with spaces instead of tabs.